week-1/Homework/mandatory/3-function-output.js

```diff
@@ -1,31 +1,39 @@
-// Add comments to explain what this function does. You're meant to use Google!
+// This function returns a number in the range of 0 (inclusive) and 1 (exclusive) and multiplies it by 10.
```

Comment:
Correct! I really like the inclusive/exclusive bits here, it's important when thinking like a programmer!